specification/compute/resource-manager/readme.md

```diff
@@ -34,7 +34,7 @@ These are the global settings for the Compute API.
 title: ComputeManagementClient
 description: Compute Client
 openapi-type: arm
-tag: package-2018-04-01
+tag: package-2018-06-01

```

Comment:
If you update this default tag to include gallery, it is very difficult to change later.  If gallery is still preview, do not update default tag.